saleor/graphql/discount/mutations.py

```diff
@@ -339,14 +342,18 @@ def clean_prices(cls, cleaned_input, voucher, errors):
             if missing_required_value or discount_value is None:
                 channels_without_value.append(cleaned_channel["channel_id"])
             # Validate value precision if it is fixed amount voucher
-            if discount_value and is_fixed_value_type:
+            if voucher.discount_value_type == DiscountValueType.FIXED:
```

Comment:
Why we drop `discount_value` from this condition?

Reply:
Because we already checked if discount_value is None at 341 and 342 lines.